AppController/lib/haproxy.rb

```diff
@@ -409,7 +383,7 @@ def self.initialize_config(connect_timeout)
     # parameters for both haproxies.
     File.open(BASE_CONFIG_FILE, 'w+') { |dest_file| dest_file.write(base_config) }
     File.open(SERVICES_BASE_FILE, 'w+') { |dest_file|
-      dest_file.write(base_config.sub('/stats', '/service-stats'))
+      dest_file.write(base_config.sub('/stats', '/services-stats'))
```

Comment:
I would prefer `service-stats` since "services stats" is grammatically incorrect.

Reply:
LOL thanks, you got it.